### PR TITLE
CRD values for Aggregate Player Tracking

### PR DESCRIPTION
**Pull request by markmandel**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug
> /kind cleanup
> /kind documentation

/kind feature

> /kind hotfix

**What this PR does / Why we need it**:
This PR is only the structure to store aggregate values for player tracking data across Fleets and GameServerSets.

**Which issue(s) this PR fixes**:

Work on #1033

**Special notes for your reviewer**:

Later PRs will implement the logic.

### CODE REVIEW

File: pkg/apis/agones/v1/common.go [hunk 1]
```diff
@@ -35,6 +35,12 @@ const (
 	ErrContainerNameInvalid     = "Container must be empty or the name of a container in the pod template"
 )
 
+// AggregatePlayerStatus stores total player tracking values
+type AggregatePlayerStatus struct {
```
Comment (pooneh-m) [inline pkg/apis/agones/v1/common.go:39]: s/Aggregate/Aggregated?

File: pkg/apis/agones/v1/fleet.go [hunk 2]
```diff
@@ -76,6 +76,11 @@ type FleetStatus struct {
 	ReservedReplicas int32 `json:"reservedReplicas"`
 	// AllocatedReplicas are the number of Allocated GameServer replicas
 	AllocatedReplicas int32 `json:"allocatedReplicas"`
+	// [Stage:Alpha]
+	// [FeatureFlag:PlayerTracking]
+	// Players are the current total player capacity and count for this Fleet
+	// +optional
+	Players *AggregatePlayerStatus `json:"players"`
```
Comment (pooneh-m) [inline pkg/apis/agones/v1/fleet.go:83]: omitempty

File: pkg/apis/agones/v1/gameserverset.go [hunk 3]
```diff
@@ -75,6 +75,11 @@ type GameServerSetStatus struct {
 	AllocatedReplicas int32 `json:"allocatedReplicas"`
 	// ShutdownReplicas are the number of Shutdown GameServers replicas
 	ShutdownReplicas int32 `json:"shutdownReplicas"`
+	// [Stage:Alpha]
+	// [FeatureFlag:PlayerTracking]
+	// Players are the current total player capacity and count for this GameServerSet
+	// +optional
+	Players *AggregatePlayerStatus `json:"players"`
```
Comment (pooneh-m) [inline pkg/apis/agones/v1/gameserverset.go:82]: omitempty

Comment (markmandel) [reply pkg/apis/agones/v1/gameserverset.go:82]: Ah yes! Nice!

File: pkg/apis/agones/v1/common.go (hunk 1 above)
Comment (markmandel) [reply pkg/apis/agones/v1/common.go:39]: Aggregated works for me. Changed.

**Comment by pooneh-m** [comment]
Nice!

**Pull request merged**